### ISSUE TITLE
Redirect to Block explorer 

### ISSUE THREAD

**Issue by edulanasca** [Quantum3-Labs/scaffold-stark-2#60]
## Issue Overview

Using [this](https://github.com/Quantum3-Labs/scaffold-stark-2/blob/bf3aea74501c1c2d08d7b38695f55887d5134bab/packages/nextjs/utils/scaffold-stark/networks.ts#L29C1-L76C2) as example 
The Block Explorer button must redirect to the starknet explorer

## Proposed Solutions or Ideas

Outline your proposed solution or share any ideas you have for discussing or resolving the issue.

**Comment by edulanasca** [comment]
Can't we use this json as a local block explorer?
<img width="537" alt="image" src="https://github.com/Quantum3-Labs/scaffold-stark-2/assets/43045722/70ed5c7a-001f-477a-8b74-f35594500664">

**Comment by 0xquantum3labs** [comment]
>Can't we use this json as a local block explorer?

No, we can either hide it in `devnet` or redirect to mainnet blockexplorer. This is a transaction receipt. not a block explorer
